doc/tutorial/00_first_example/index.md

```diff
@@ -1,39 +0,0 @@
-# A first example {#tutorial_first_example}
-
-[TOC]
-
-This tutorial walks you through a small SeqAn3 program. It is intended to give you a short overview of what to expect
-in the other tutorials and how to use this documentation.
-
-<table>
-<tr>  <th> Difficulty             </th>  <td> Easy                   </td>   </tr>
-<tr>  <th> Duration               </th>  <td> 30-60 min              </td>   </tr>
-<tr>  <th> Prerequisite tutorials </th>  <td> @ref setup             </td>   </tr>
-<tr>  <th> Recommended reading    </th>  <td>                        </td>   </tr>
-</table>
-
----
-
-*Every page in the tutorials begins with this section. It is recommended you do the "prerequisite tutorials" before
-the current one. You should also have a look at the links provided in "recommended reading" and maybe keep them open
-in separate tabs/windows as reference.*
-
-***These tutorials try to briefly introduce C++ features not well known, however they do not teach programming in C++!
-If you know how to program in another language, but are not familiar with C++ and/or the significant
-changes in the language in recent years, we recommend the following resources:***
-
-  * "A Tour of C++", Second Edition, Bjarne Stroustrup, 2018
-
-# First head line
-
-`some inline code`
-
-## A second head line
-
-\note This is a note.
```

Comment:
Mhm, I'll make a card to make a wiki post where all the custom commands are explained (after the pmsb).

Reply:
Thank you, that's a great idea!